ephios/core/templates/core/userprofile_form.html

```diff
@@ -94,17 +94,18 @@ <h3 class="card-title">{% translate "Qualifications" %}</h3>
                 {% endescapescript %}
             </script>
         </div>
-            <a role="button" class="btn btn-secondary"
-               href="{% url "core:userprofile_list" %}">{% trans "Cancel" %}</a>
-            {% if userprofile %}
-                <a class="btn btn-outline-secondary text-nowrap" href="{% url "core:userprofile_password_reset" userprofile.pk %}">
-                    <span class="d-none d-md-inline">{% trans "Reset password" %}</span></a>
```

Comment:
What was the reason for having the span here @gritfessel? :)

Reply:
Good question, probably a copy paste issue